src/Transaction/CustomParameter.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Buckaroo\Transaction;
+
+/**
+ * This class holds information about the custom parameter of the transaction.
+ */
+class CustomParameter
```

Comment:
I think this should be named Parameter as it's also used for additional parameters